doxygen/general_pages/CppSetupGuide.dox

```diff
@@ -91,6 +91,33 @@ cd vcpkg
   - Open `MeshLib/source/MeshLib.sln` in Visual Studio.
   - Build the solution and run the application.
 
+#### Using the CMake build system
+
+MeshLib can also be build with CMake. The supported generators are: Ninja, NMake, and jom.
+
+\note Visual Studio Code uses the Visual Studio generator by default, but it is unsupported. You can change the generator in the CMake Tools extension's settings.
```

Comment:
Why unsupported?

Reply:
It leads to various build errors and incorrect binaries. We haven't investigated the actual reasons, but I suppose it might conflict with existing Visual Studio project files.